build.sbt

```diff
@@ -106,6 +119,23 @@ lazy val `anorm-akka` = (project in file("akka"))
       "com.typesafe.akka" %% "akka-stream-contrib" % "0.6" % Test)
   )).dependsOn(anorm)
 
+lazy val pgVer = sys.env.get("POSTGRES_VERSION").getOrElse("9.4.1212")
+
+val jsonVer = Def.setting[String] {
```

Comment:
Can we just always depend on 2.6.x? I thought we were releasing 2.6.0 so we can break binary compatibility.